konrad/radiation/rrtmg.py

```diff
@@ -182,6 +196,23 @@ def update_cloudy_radiative_state(self, cloud, state0, sw=True):
         for varname in props + sw_props if sw else props + lw_props:
             state0[varname] = cloud[varname]
 
+#    def update_aerosol_radiative_properties(self, aerosol, state_sw, state_lw):
+#        state_sw['shortwave_optical_thickness_due_to_aerosol'] = aerosol.optical_thickness_due_to_aerosol_sw
+#        state_sw['single_scattering_albedo_due_to_aerosol'] = aerosol.single_scattering_albedo_aerosol_sw
+#        state_sw['aerosol_asymmetry_parameter'] = aerosol.asymmetry_factor_aerosol_sw
+#        state_lw['longwave_optical_thickness_due_to_aerosol'] = aerosol.optical_thickness_due_to_aerosol_lw
+
+        return
+
+    def update_aerosol_radiative_properties(self, atmosphere, aerosol, state_sw, state_lw):
+        aerosol.update_aerosols(1, atmosphere)
```

Comment:
As far as I understand, the aerosol state is already updated in ``konrad.core``. Is there a reason to update it again?

Reply:
Thanks, I will look into this and make a new pull request once the aerosol class is finished.